### PR TITLE
Add checks for contract fees

### PR DESCRIPTION
**Pull request by kwingram25**
Still unclear how fees are calculated, reached out to Sergei for help!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,6 @@
   ],
   "resolutions": {
     "@polkadot/api": "^0.82.0-beta.6",
-    "@polkadot/api-contract": "^0.82.0-beta.6",
```
Comment (jacogr) [inline package.json]: We should just re-add this.

File: packages/ui-signer/src/Checks/Call.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+type Props = I18nProps & {
+  endowment: BN | Compact,
+  fees: DerivedContractFees,
+  onChange: (fees: ExtraFees) => void
+};
+
+type State = ExtraFees & {};
+
+export class Call extends React.PureComponent<Props, State> {
```
Comment (jacogr) [inline packages/ui-signer/src/Checks/Call.tsx]: Would probably make more sense as ContractCall? (Call can also mean, well, `call` :))

File: packages/ui-signer/src/Checks/Call.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+    return update;
+  }
+
+  render () {
+    return null;
+  }
+}
+
+export default translate(Call);
```
Comment (jacogr) [inline packages/ui-signer/src/Checks/Call.tsx]: We can remove the translate wrapper, no text here.

File: packages/ui-signer/src/Checks/Deploy.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+type Props = I18nProps & {
+  endowment: BN | Compact,
+  fees: DerivedContractFees,
+  onChange: (fees: ExtraFees) => void
+};
+
+type State = ExtraFees & {};
+
+export class Deploy extends React.PureComponent<Props, State> {
```
Comment (jacogr) [inline packages/ui-signer/src/Checks/Deploy.tsx]: ContractDeploy? (Just to align with Call)

File: packages/ui-signer/src/Checks/Deploy.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+    return update;
+  }
+
+  render () {
+    return null;
+  }
+}
+
+export default translate(Deploy);
```
Comment (jacogr) [inline packages/ui-signer/src/Checks/Deploy.tsx]: As above, redundant.

File: package.json (hunk 1 above)
Comment (kwingram25) [reply package.json]: It's not included yet as part of /api?

**Comment by kwingram25** [comment]
Fees are correct afaik!

**Pull request merged**